tests/CommandsTest.php

```diff
@@ -336,6 +338,50 @@ public function testWorkingDirectoryToken()
         $this->assertContains('/tests/sandbox', $runner->getConfig()->get('working_dir'));
     }
 
+    /**
+     * Test the user config.
+     */
+    public function testUserConfigFile()
+    {
+        $fixtureName = 'userconfig.yml';
+
+        $settings = [
+            'TASKRUNNER_CONFIG' => __DIR__ . '/fixtures/' . $fixtureName,
+        ];
+
+        // Convert the array into a proper string.
+        $envSettings = implode(' ', array_map(
+            function ($value, $key) {
+                return sprintf('%s=%s', $key, $value);
+            },
+            $settings,
+            array_keys($settings)
+        ));
+
+        // Create a runner.
+        $input = new StringInput('list --working-dir=' . $this->getSandboxRoot());
+        $runner = new TaskRunner($input, new NullOutput(), $this->getClassLoader());
+        $runner->run();
+
+        // Extract a value from the default configuration.
+        $drupalRoot = $runner->getConfig()->get('drupal.root');
+
+        // Add the environment setting.
+        putenv($envSettings);
```

Comment:
Can we simplify with:
```
        // Add the environment setting.
        putenv('TASKRUNNER_CONFIG=' . __DIR__ . '/fixtures/userconfig.yml');
```
and removing
```
         $fixtureName = 'userconfig.yml';
         $settings = [
            'TASKRUNNER_CONFIG' => __DIR__ . '/fixtures/' . $fixtureName,
        ];
         // Convert the array into a proper string.
        $envSettings = implode(' ', array_map(
            function ($value, $key) {
                return sprintf('%s=%s', $key, $value);
            },
            $settings,
            array_keys($settings)
        ));
```

Reply:
That was done just in case if we want to test multiple variables. But ok I will remove it.

Reply:
Fixed.